app/controllers/admin/content_controller.rb

```diff
@@ -113,6 +113,19 @@ def autosave
     render :text => nil
   end
 
+  def merge
+# find the ID of the article the admin submitted
+    @article1 = Article.find(params[:merge_with])
```

Comment:
I'm curious why you chose one to be an instance variable and the other to be a local variable